src/main/app/src/app/dashboard/dashboard.component.ts

```diff
@@ -39,7 +38,7 @@ export class DashboardComponent implements OnInit {
     new DashboardCard(
       DashboardCardId.MIJN_TAKEN_NIEUW,
       DashboardCardType.TAKEN,
-      SignaleringType.TAAK_OP_NAAM,
+      "TAAK_OP_NAAM",
```

Comment:
This is a shame, using static strings where they used to be enums... Can't we at least make them constant values?

Reply:
Kind off, but that is how they currently come from the generated types. It is still completely type safe though, so they are half-magic-strings

![image](https://github.com/user-attachments/assets/7f4d34e6-7f34-4c29-a16d-683c006f8fa6)

Reply:
☝️ They are type save as the `DashboardCard` does require the type to be of a specific type

```typescript
constructor(
    id: DashboardCardId,
    type: DashboardCardType,
    signaleringType?: GeneratedType<"RestSignaleringInstellingen">["type"],
  )
```